{"name": "test_sample01[123-123--]", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: Service drivers/chromedriver unexpectedly exited. Status code was: -9", "trace": ".0 = <list_iterator object at 0x102acb910>\n\n>   kwargs = {arg: kwargs[arg] if arg in kwargs else request.getfixturevalue(arg) for arg in args}\n\n../../../Automation/lib/python3.11/site-packages/pytest_bdd/scenario.py:204: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n../../../Automation/lib/python3.11/site-packages/_pytest/fixtures.py:541: in getfixturevalue\n    fixturedef = self._get_active_fixturedef(argname)\n../../../Automation/lib/python3.11/site-packages/_pytest/fixtures.py:570: in _get_active_fixturedef\n    self._compute_fixture_value(fixturedef)\n../../../Automation/lib/python3.11/site-packages/_pytest/fixtures.py:647: in _compute_fixture_value\n    fixturedef.execute(request=subrequest)\n../../../Automation/lib/python3.11/site-packages/_pytest/fixtures.py:1079: in execute\n    result = ihook.pytest_fixture_setup(fixturedef=self, request=request)\n../../../Automation/lib/python3.11/site-packages/pluggy/_hooks.py:501: in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n../../../Automation/lib/python3.11/site-packages/pluggy/_manager.py:119: in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n../../../Automation/lib/python3.11/site-packages/_pytest/setuponly.py:36: in pytest_fixture_setup\n    return (yield)\n../../../Automation/lib/python3.11/site-packages/_pytest/fixtures.py:1132: in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n../../../Automation/lib/python3.11/site-packages/_pytest/fixtures.py:906: in call_fixture_func\n    fixture_result = next(generator)\nsteps/conftest.py:30: in init_driver\n    driver = webdriver.Chrome(service=service,options=options)\n../../../Automation/lib/python3.11/site-packages/selenium/webdriver/chrome/webdriver.py:45: in __init__\n    super().__init__(\n../../../Automation/lib/python3.11/site-packages/selenium/webdriver/chromium/webdriver.py:50: in __init__\n    self.service.start()\n../../../Automation/lib/python3.11/site-packages/selenium/webdriver/common/service.py:102: in start\n    self.assert_process_still_running()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.chrome.service.Service object at 0x102b068d0>\n\n    def assert_process_still_running(self) -> None:\n        \"\"\"Check if the underlying process is still running.\"\"\"\n        return_code = self.process.poll()\n        if return_code:\n>           raise WebDriverException(f\"Service {self._path} unexpectedly exited. Status code was: {return_code}\")\nE           selenium.common.exceptions.WebDriverException: Message: Service drivers/chromedriver unexpectedly exited. Status code was: -9\n\n../../../Automation/lib/python3.11/site-packages/selenium/webdriver/common/service.py:115: WebDriverException"}, "description": "../features/sample.feature: Verify the login functionality", "parameters": [{"name": "_pytest_bdd_example", "value": "{'Username': '123', 'Password': '123'}"}, {"name": "Username", "value": "''"}, {"name": "Password", "value": "''"}], "start": 1711774194891, "stop": 1711774209955, "uuid": "7e5d2e34-c08d-444c-baad-6819333f986e", "historyId": "aa800b01451927b0f97456ee7ee977c5", "testCaseId": "8781a52a01f4bb11b862418b60a465e1", "fullName": "steps.test_sample#test_sample01", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('Username', 'Password')"}, {"name": "tag", "value": "sample01"}, {"name": "parentSuite", "value": "steps"}, {"name": "suite", "value": "test_sample"}, {"name": "host", "value": "Saravanas-MacBook-Air.local"}, {"name": "thread", "value": "2349-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "steps.test_sample"}]}